lisa/operating_system.py

```diff
@@ -789,6 +862,15 @@ class Fedora(Linux):
     # microsoft-azure-rhel8-eus  Microsoft Azure RPMs for RHEL8 Extended Update Support
     _fedora_repository_info_pattern = re.compile(r"(?P<id>\S+)\s+(?P<name>\S.*\S)\s*")
 
+    # ex: dpdk-20.11-3.el8.x86_64
+    _fedora_version_splitter_regex = re.compile(
```

Comment:
try named groups, it should be much easier to self-explain.

Reply:
I learn something new every day 😂 will change